src/screens/ChangeWalletSheet.js

```diff
@@ -362,7 +358,7 @@ const ChangeWalletSheet = () => {
           <Divider color={colors.rowDividerExtraLight} inset={[0, 15]} />
         )}
       </Column>
-      <EditButton onPress={toggleEditMode}>
+      <EditButton onPress={() => setEditMode(e => !e)}>
```

Comment:
Don't you wanna keep the useCallback around here?

Reply:
I think it's not worth memoizing